clients/github-graphql/build.gradle.kts

```diff
@@ -34,6 +34,11 @@ graphql {
         schemaFile = file("${project.projectDir}/src/main/assets/schema.graphql")
         queryFileDirectory = "${project.projectDir}/src/main/assets/"
         serializer = GraphQLSerializer.KOTLINX
+
```

Comment:
Commit message:
* Nit: "token**s**"
* Any link to the release notes?

Reply:
All done.